spec/vendor_spec.rb

```diff
@@ -0,0 +1,39 @@
+require './lib/vendor'
+require './lib/item'
+
+RSpec.describe Vendor do
+    before(:each) do
+        @vendor = Vendor.new("Rocky Mountain Fresh")
+        @item1 = Item.new({name: 'Peach', price: "$0.75"})
+        @item2 = Item.new({name: 'Tomato', price: "$0.50"})
+    end
+    
+    it 'exists' do
+        expect(@vendor).to be_a(Vendor)
+        expect(@vendor.name).to eq("Rocky Mountain Fresh")
+    end
+
+    it 'has an inventory' do
+        expect(@vendor.inventory).to eq({})
+    end
+
+    it 'checks stocks of items' do
+        expect(@vendor.check_stock(@item1)).to eq(0)
+    end
+
+    it 'stocks items in inventory' do
```

Comment:
These tests are failing - we're getting an object back that looks like `+#<Item:0x00007f9a182d3de0 @name="Peach", @price=0.75> => {:quantity=>30}` - I'd like to see something more like `<Item:0x00007f7f2c2cfb48 @name="Tomato", @price=0.5> => 10`. What you did here works but doesnt quite follow the interaction pattern.